ios/Classes/ClickstreamFlutterPlugin.swift

```diff
@@ -56,8 +56,7 @@ public class ClickstreamFlutterPlugin: NSObject, FlutterPlugin {
                 .withSessionTimeoutDuration(arguments["sessionTimeoutDuration"] as! Int64)
                 .withCompressEvents(arguments["isCompressEvents"] as! Bool)
                 .withAuthCookie(arguments["authCookie"] as! String)
-            if arguments["globalAttributes"] != nil {
-                let attributes = arguments["globalAttributes"] as! [String: Any]
+            if let attributes = arguments["globalAttributes"] as? [String: Any] {
```

Comment:
is it covered by e2e test?

Reply:
Flutter SDK e2e testing is not supported yet,  swift SDK this [test case](https://github.com/awslabs/clickstream-swift/blob/main/Tests/ClickstreamTests/SDKInitialTest.swift#L69-L85) have verified initial SDK without global attributes, and in this example app I also added an [case](https://github.com/awslabs/clickstream-flutter/pull/11/files#diff-72fd825ad17ce7800a2078e06f0dff1e12836ee59760465e78d9a5d222cebd15R61) for verify initialize flutter SDK without global attributes.